skinStyles/extensions/StructuredNavigation/ext.structuredNav.NavigationView.less

```diff
@@ -0,0 +1,49 @@
+/*
+ * Citizen
+ *
+ * SkinStyles for StructuredNavigation
+ * Module: ext.structuredNav.NavigationView.styles
+ * Version: REL1_39 55e2ec0
+ *
+ * Date: 2024-09-06
+*/
+
+/* ext.structuredNav.NavigationView.less */
+.mw-structurednav-navigation-container {
+	background-color: var(--color-surface-2);
+	border: 2px solid var(--border-color-base);
```

Comment:
You can just set the `border-color` to `var( --border-color-subtle )`. This is what was used on the [original style](https://github.com/wikimedia/mediawiki-extensions-StructuredNavigation/blob/be436f74c8674c1d61142133f7aebf2639c48f3d/resources/styles/variables.less#L5).